pkg/status/combinedstatus-resolution.go

```diff
@@ -640,13 +671,22 @@ func handleCombinedFields(scName string, scData *statusCollectorData) *v1alpha1.
 	return &namedStatusCombination
```

Comment:
(I finally figured out that the purpose of this method is to compute the intended value of one NamedStatusCombination.)

The logic in this method has some problems (not introduced in this PR). One problem is that a NamedStatusCombination should have three kinds of columns, for the GroupBy, CombinedFields, and Select items in the StatusCollectorSpec.

There are several additional problems in the aggregation logic in this method. Perhaps I will try to update the documentation of what this feature is intended to do.

Reply:
Please see #2325 .